go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/dave/patsy
+
+go 1.15
```

Comment:
go keeps automatically adding `go 1.15` (based on the version you're using) if you don't set it yourself, I think we could set the minimum version here?